coordinator/src/main/java/org/apache/uniffle/coordinator/web/servlet/NodesServlet.java

```diff
@@ -38,19 +38,31 @@ public NodesServlet(CoordinatorServer coordinator) {
 
   @Override
   protected Response<List<ServerNode>> handleGet(HttpServletRequest req, HttpServletResponse resp) {
-    List<ServerNode> serverList = coordinator.getClusterManager().getServerList(Collections.EMPTY_SET);
-    String id = req.getParameter("id");
-    String status = req.getParameter("status");
-    serverList = serverList.stream().filter((server) -> {
-      if (id != null && !id.equals(server.getId())) {
-        return false;
-      }
-      if (status != null && !server.getStatus().toString().equals(status)) {
-        return false;
-      }
-      return true;
-    }).collect(Collectors.toList());
-    Collections.sort(serverList, Comparator.comparing(ServerNode::getId));
-    return Response.success(serverList);
+    String type = req.getParameter("type");
```

Comment:
Why do we need a new parameter? Could we use `status`?

Reply:
Do you miss this thread? @yl09099

Reply:
And what if the parameters contains status and id?

Reply:
It has been modified, please review it again